exporter-errors/fatal-errors.json

```diff
@@ -62,6 +62,12 @@
       "canUserFix": false,
       "kbLinkId": ""
     },
+    "InvalidUserDefinedSchemaMappingJSON" : {

```

Comment:
Can we provide any additional information to help the user fix the problem? It's listed as user fixable, but the message doesn't give any details...

Reply:
We will be updating the description dynamically when there is a error in the validation method of the JSON .